### DISCUSSION TITLE
How do we get the ELF binary for the terraform ops_images if we're using a package for the binary?

### DISCUSSION THREAD

**Discussion by radiosilence** [nanovms/ops#1595] [Q&A]
So the situation would be, our application is simply a `config.json` and some other files, that relies upon a package to provide the binary.

ops_images requires a key of name= which seems to be a binary it looks for, but this of course isn't there as the binary itself is in the package, not our directory.

**Comment by eyberg** [comment]
a lot of people don't really need terraform (instead just using 'image create' and 'instance create') can usually suffice, but some people want it so it fits in with their other terraform

there does appear to be some package support available though: https://github.com/nanovms/terraform-provider-ops/tree/master/examples/package (although it might need to be updated for namespaces)

**Reply by eyberg** [reply]
https://github.com/nanovms/terraform-provider-ops/issues/10
